plugins/woocommerce-beta-tester/src/tools/data/actions.js

```diff
@@ -117,6 +117,8 @@ export function* resetOnboardingWizard() {
 			woocommerce_price_decimal_sep: '.',
 			woocommerce_price_num_decimals: '2',
 			woocommerce_coming_soon: 'no',
+			woocommerce_weight_unit: 'kg',
+			woocommerce_dimension_unit: 'cm',
```

Comment:
The default weight and dimension units for the US are 'lbs' and 'in':

```suggestion
			woocommerce_weight_unit: 'lbs',
			woocommerce_dimension_unit: 'in',
```

Reply:
oops 🥶